tests/hubapi/mvcgi/README

```diff
@@ -9,3 +9,4 @@ HUB_API_HOST=http://localhost:7080 py.test
 A vanilla Biothings Studio is expected with these tests:
 $ docker pull biothings/biothings-studio
 $ docker run --rm --name test_studio -p 8080:8080 -p 7022:7022 -p 7080:7080 -p 9001:9000 -p 8000:8000 -d biothings/biothings-studio
+
```

Comment:
@remoteeng00 should all your changes to this `mvcgi` folder moved to `demohub` folder? and leaves mvcgi folder unchanged?